src/coreclr/jit/gentree.cpp

```diff
@@ -21048,7 +21102,18 @@ GenTree* Compiler::gtNewSimdCndSelNode(
 
 #if defined(TARGET_XARCH)
     assert((simdSize != 32) || compIsaSupportedDebugOnly(InstructionSet_AVX));
-    intrinsic = (simdSize == 32) ? NI_Vector256_ConditionalSelect : NI_Vector128_ConditionalSelect;
+    if (simdSize == 32)
+    {
+        intrinsic = NI_Vector128_ConditionalSelect;
```

Comment:
Intentional that this changed from the 256 to 128 variant?

Reply:
That's a bug. Will fix it!